tests/src/test/scala/actionContainers/BasicActionRunnerTests.scala

```diff
@@ -19,14 +19,32 @@ package actionContainers
 
 import org.junit.runner.RunWith
 import org.scalatest.junit.JUnitRunner
-
 import spray.json.DefaultJsonProtocol._
 import spray.json._
 
 @RunWith(classOf[JUnitRunner])
 trait BasicActionRunnerTests extends ActionProxyContainerTestUtils {
   def withActionContainer(env: Map[String, String] = Map.empty)(code: ActionContainer => Unit): (String, String)
 
+  /**
+   * Runs tests for actions which receive an empty initializer (no source or exec).
+   * @param checkResultInLogs should be true iff the result of the action is expected to appear in stdout or stderr
+   */
+  def testNoSourceOrExec(checkResultInLogs: Boolean = true) = {
```

Comment:
Seems like checkResultInLogs is not used....should it be removed?

Reply:
updated to remove the relevant code.